docs/adapters/index.md

```diff
@@ -12,6 +12,7 @@ These adapters are included in the Juttle runtime and can be used in Juttle prog
 
 * [file](../adapters/file.md)
 * [http](../adapters/http.md)
+* [io](../adapters/io.md)
```

Comment:
I could live with `read stdio` and `write stdio`. That would enable `write stdio -stream 'stderr'` as @go-oleg suggested above.